tools/BUILD

```diff
@@ -0,0 +1,13 @@
+py_runtime(
+	name = "python2.7.6",
+	files = [],
+	interpreter_path = "/usr/bin/python",
```

Comment:
The flag name here seems overly specific, right? For any given user we are not guaranteed that bin/python == 2.7.6. In fact, it may be py3, right?

Reply:
the container we are currently using uses these python versions. I am not sure what the plan of tensorflow/tensorflow:nightly-custom-op is and how they update the versions. But yea i can change this to specify the version.

Reply:
Moritz, have you read tensorflow script and know how it solve the problem ? https://github.com/tensorflow/tensorflow/blob/f089b3180ec7ddecd503d6d499cd6977c22b8f11/configure.py#L202

Reply:
Now i have. Thank you for the hint :+1: